### PR TITLE
fix #7351 chore(project): add make command for gcloud auth

### LINKED ISSUE #7351: Add a make command for gcloud auth
Now that we've migrated over to using the gcloud auth cli, we can probably add a make command that wraps all the steps, particularly because it seems you need to periodically reauthorize.

### PR DESCRIPTION
**Pull request by mikewilli**
Because

* certain local dev requires authenticating to GCP
* GCP expires authentication periodically

This commit

* adds a helpful `make auth` command to run the required commands

**TESTING INFO**
- Follow [Google Credentials for Jetstream](https://github.com/mozilla/experimenter/blob/620d3a59dbd8ed04e8d90ff77334cc29ba2a29a1/README.md#google-credentials-for-jetstream) in the README

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -62,6 +62,10 @@ nginx/cert.pem: nginx/key.pem
 secretkey:
 	openssl rand -hex 24
 
+auth:
```
Comment (jaredlockhart) [inline Makefile:65]: I might name this something more explicitly clear that it's for gcloud like `gcloud_auth` or something?

**Review by jaredlockhart — CHANGES_REQUESTED** [review]
Don't forget to do the thing I literally always forget to do.... update the README!

Also awesome thank you @mikewilli this whole process is so much easier now 🙏

File: README.md [hunk 2]
```diff
@@ -197,8 +197,7 @@ On certain pages an API endpoint is called to receive experiment analysis data f
       - If needed, ask in `#nimbus-dev` for a project admin to grant `storage.objects.list` permissions on the `moz-fx-data-experiments` project
 
 1. Authorize CLI with your account
-    - `gcloud auth login`
-    - `gcloud auth application-default login`
+    - `make auth`
```
[the remark this answers is not in the document]
Comment (mikewilli) [reply README.md]: :facepalm: ...fixed

**Review by jaredlockhart — APPROVED** [review]
Looks good to me and works locally, I'd like someone else to try it too @yashika-khurana @eliserichards ?

**Comment by yashika-khurana** [comment]
> Looks good to me and works locally, I'd like someone else to try it too @yashika-khurana @eliserichards ?

I am testing it now 😊

**Comment by yashika-khurana** [comment]
Worked for me. thanks @mikewilli 😊

**Review by yashika-khurana — APPROVED** [review]

**Pull request merged**